app/controllers/welcome_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
     @partners = @current_event.sponsors
     @gallery_photos = @current_event.gallery_photos.limit(8)
     @more_photos = @current_event.gallery_photos.count > 8
+    @event_previews = @current_event.event_previews
```

Comment:
For the welcome page, I think you don't need to load the event previews because we will only display it on the individual event page